app/src/main/java/de/danoeh/antennapod/activity/VideoplayerActivity.java

```diff
@@ -149,34 +143,31 @@ protected void setupGUI() {
         if (isSetup.getAndSet(true)) {
             return;
         }
+        VideoplayerActivityBinding viewBinding = VideoplayerActivityBinding.inflate(getLayoutInflater());
```

Comment:
Shouldn't this use the class variable `viewBinding` here instead of creating a new one?